tfjs-backend-webgpu/src/kernels/FromPixels_utils/from_pixels_webgpu.ts

```diff
@@ -70,8 +70,8 @@ export class FromPixelsProgram implements WebGPUProgram {
     return `
       [[binding(1), group(0)]] var src: ${textureType};
 
-      ${getMainHeaderStringWgsl(this.workGroupSize)} {
-        ${getGlobalIndexStringWgsl(this.workGroupSize)}
+      ${getMainHeaderStringWgsl()} {
+        ${getGlobalIndexStringWgsl()}
```

Comment:
Not sure if it's still necessary to call `${getGlobalIndexStringWgsl()}` rather than directly using `let index = getGlobalIndex(globalId, localId);`. The latter is more intuitive.

Reply:
Yes, use getGlobalIndex(globalId, localId) is more intuitive, but not easy to change.  And use ${getGlobalIndexStringWgsl()} is more easy to change for all shaders, but not intuitive. WDYT?